urls.py

```diff
@@ -13,7 +13,10 @@
     path('', views.webgui_test, name='index'),
```

Comment:
*Annoying comment* : try to use infinitive for commit messages :) (and first letter in big caps)